### PR TITLE
send telemetry to honeycomb

### PR DESCRIPTION
**Pull request by imykolenko**

### CODE REVIEW

File: backend/src/main/kotlin/xyz/funkybit/Main.kt [hunk 1]
```diff
@@ -5,15 +5,15 @@ import xyz.funkybit.apps.api.ApiApp
...
+import xyz.funkybit.core.telemetry.telemetry
 import kotlin.system.exitProcess
 
 private val logger = KotlinLogging.logger {}
 
 fun main(args: Array<String>) {
     logger.info { "Starting with args: ${args.joinToString(" ")}" }
 
-    val telemetry = sentryTelemetry
+    val telemetry = telemetry
```
Comment (antonyshchenko) [inline backend/src/main/kotlin/xyz/funkybit/Main.kt:16]: Maybe just get rid of the `telemetry` val here

File: src/main/kotlin/xyz/funkybit/LocalMain.kt [hunk 2]
```diff
@@ -17,7 +17,7 @@ import kotlin.system.exitProcess
 
 fun main(args: Array<String>) {
     val logger = KotlinLogging.logger {}
-    val telemetry = sentryTelemetry
+    val telemetry = telemetry
```
Comment (antonyshchenko) [inline src/main/kotlin/xyz/funkybit/LocalMain.kt:20]: And here

**Review by antonyshchenko — APPROVED** [review]
Looks great!

File: backend/src/main/kotlin/xyz/funkybit/Main.kt (hunk 1 above)
Comment (imykolenko) [reply backend/src/main/kotlin/xyz/funkybit/Main.kt]: sure, makes sense, after renaming local var is obsolete.

**Pull request merged**